src/Microsoft.CodeQuality.Analyzers/CSharp/QualityGuidelines/CSharpRemoveEmptyFinalizersAnalyzer.cs

```diff
@@ -13,39 +14,79 @@ internal class CSharpRemoveEmptyFinalizersAnalyzer : AbstractRemoveEmptyFinalize
     {
         protected override bool IsEmptyFinalizer(SyntaxNode methodBody, CodeBlockAnalysisContext analysisContext)
         {
-            var destructorDeclaration = (DestructorDeclarationSyntax)methodBody;
+            // TODO: Switch the analyzer to be IOperation based to avoid handling different syntactic constructs.
```

Comment:
💡 Should move this to be an issue instead of a comment

Reply:
Filed https://github.com/dotnet/roslyn-analyzers/issues/1964 and remove the comment.